test/logics/radiant-v2/borrow.test.ts

```diff
@@ -0,0 +1,73 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { claimToken, getChainId, snapshotAndRevertEach } from '@protocolink/test-helpers';
+import * as common from '@protocolink/common';
+import * as core from '@protocolink/core';
+import { expect } from 'chai';
+import * as helpers from './helpers';
+import hre from 'hardhat';
+import * as radiantv2 from 'src/logics/radiant-v2';
+
+describe('arbitrum: Test RadiantV2 Borrow Logic', () => {
+  let chainId: number;
+  let users: SignerWithAddress[];
+
+  before(async () => {
+    chainId = await getChainId();
+    const [, user1, user2] = await hre.ethers.getSigners();
+    users = [user1, user2];
+    await claimToken(chainId, user1.address, radiantv2.arbitrumTokens.USDC, '20000');
```

Comment:
Should we follow the same style to use the `arbitrumTokens` from test-helpers?

Reply:
I think the `arbitrumTokens` from test-helpers are mainly for the most common tokens like `USDC`, `USDT`, and `WETH`. 

However, there might be cases where we need to include specific tokens (such as `aToken` or `rToken`) for testing that are not included in `arbitrumTokens` from test-helpers. In such cases, I believe it would be more appropriate to use `radiantv2.arbitrumTokens.USDC` when we have to get `token`.